index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+    integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <header id="header">
+    <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" id="header-img"
+      alt="">
+    <nav id="nav-bar">
+      <ul>
+        <li><a class="nav-link" href="#features">Features</a></li>
+        <li><a class="nav-link" href="#how-it-works">How it works</a></li>
+        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+      <h1>Handcrafted, home-made masterpieces</h1>
+      <input id="email" type="email" placeholder="Enter you email address"> <br>
+      <input id="submit" type="submit" value="GET STARTED">
+    </form>
+    <section id="features">
+      <div>
+        <i class="fa fa-3x fa-fire"></i>
+        <h1>Premium Materials</h1>
+        <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your
+          purchase.</p>
+      </div>
+      <div>
+        <i class="fa fa-3x fa-truck"></i>
+        <h1>Fast Shipping</h1>
+        <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if
+          you are
+          not satisfied.</p>
+      </div>
+      <div>
+        <i class="fa fa-3x fa-battery-full"></i>
+        <h1>Quality Assurance</h1>
+        <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the
+          pitch
+          of your
+          instrument.</p>
+      </div>
```

Comment:
[OPTIONAL]
The classes fa, fa-3x, fa-fire, and fa-battery-full weren't used in this file. Remember to keep your code clean!

Reply:
> [OPTIONAL] The classes fa, fa-3x, fa-fire, and fa-battery-full weren't used in this file. Remember to keep your code clean!

- [ ] kindly consider to phrased the comments politely, as suggestions instead of orders. For example:

* This is a polite suggestion: *"I think it would be better to use semantic tags"*

Reply:
> [OPTIONAL] The classes fa, fa-3x, fa-fire, and fa-battery-full weren't used in this file. Remember to keep your code clean!

- [ ] kindly make your comments are clear and easy to understand. For Example:

* This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*